### PR TITLE
Added New Project Phonebook

### PR DESCRIPTION
**Pull request by MarinaNikole**

### CODE REVIEW

File: src/main/java/ContactsPage.java [hunk 1]
```diff
@@ -1,42 +1,61 @@
...
-    @FindBy(css = "b")
-    private WebElement foundContact;
+
+    @FindBy(css = "[class=\"nav-tabs bg-secondary text-white d-flex align-items-center p-2 nav\"]")
+    private WebElement contactsListElement;
+
+    @FindBy(id= "input-search-contact")
+    private WebElement searchInputField;
+
+    @FindBy(xpath = "(//div[@id=\"contacts-list\"]//b)[1]")
```
Comment (antoncheprasov) [inline src/main/java/ContactsPage.java:23]: Is it possible to find a better selector?

File: src/main/java/ContactsPage.java [hunk 2]
```diff
@@ -1,42 +1,61 @@
...
+    private WebElement newContatNameFromContactsList;
+
+    @FindBy(xpath = "(//div[@id=\"contacts-list\"]//button)[1]")
```
Comment (antoncheprasov) [inline src/main/java/ContactsPage.java:26]: Is it possible to find a better selector?

File: src/main/java/ContactsPage.java [hunk 3]
```diff
@@ -1,42 +1,61 @@
...
-    private WebElement foundContact;
+
+    @FindBy(css = "[class=\"nav-tabs bg-secondary text-white d-flex align-items-center p-2 nav\"]")
+    private WebElement contactsListElement;
+
+    @FindBy(id= "input-search-contact")
+    private WebElement searchInputField;
+
+    @FindBy(xpath = "(//div[@id=\"contacts-list\"]//b)[1]")
+    private WebElement newContatNameFromContactsList;
```
Comment (antoncheprasov) [inline src/main/java/ContactsPage.java:24]: a missing "c" in word Contact

File: src/main/java/ContactsPage.java [hunk 4]
```diff
@@ -1,42 +1,61 @@
...
 
-    public boolean contactsPageIsDisplayed() {
-        try {
-            contactsPage.isDisplayed();
-            return true;
-        }
-        catch (NoSuchElementException e){
-            return false;
-        }
+    public void clickOnNewContactNameLinkFromContactsList(){
```
Comment (antoncheprasov) [inline src/main/java/ContactsPage.java:49]: too long name for method

File: src/main/java/RemoveContactDialog.java [hunk 5]
```diff
@@ -0,0 +1,28 @@
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class RemoveContactDialog extends BasePage{
```
Comment (antoncheprasov) [inline src/main/java/RemoveContactDialog.java:5]: its better to name this method "DeleteContactDialog"

File: src/main/java/RemoveContactDialog.java [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+    public RemoveContactDialog(WebDriver driver) {
+        super(driver);
+    }
+
+    @FindBy(id= "check-box-remove-contact")
+    private WebElement checkboxDelitionConfirm;
+
+    @FindBy(id= "submit-remove")
+    private WebElement removeContactButton;
```
Comment (antoncheprasov) [inline src/main/java/RemoveContactDialog.java:14]: rename to "deleteContactButton"

File: src/main/java/ContactsPage.java (hunk 1 above)
Comment (MarinaNikole) [reply src/main/java/ContactsPage.java:23]: Probably is possible

File: src/main/java/ContactsPage.java (hunk 2 above)
Comment (MarinaNikole) [reply src/main/java/ContactsPage.java:26]: Of course

File: src/main/java/ContactsPage.java (hunk 3 above)
Comment (MarinaNikole) [reply src/main/java/ContactsPage.java:24]: I`ll fix.

File: src/main/java/ContactsPage.java (hunk 4 above)
Comment (MarinaNikole) [reply src/main/java/ContactsPage.java:49]: I agree

File: src/main/java/RemoveContactDialog.java (hunk 5 above)
Comment (MarinaNikole) [reply src/main/java/RemoveContactDialog.java:5]: I`ll fix.

File: src/main/java/RemoveContactDialog.java (hunk 6 above)
Comment (MarinaNikole) [reply src/main/java/RemoveContactDialog.java:14]: I`ll fix.

**Pull request merged**